_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,433 @@
+---
+id: 9bd38c
+name: Content has non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when content is identified through a visual reference, there are also non-visual references identifying the same content.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Daniël Strik
+    - Jean-Yves Moyen
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [text node][] that is either [visible][] or [included in the accessibility tree][].
+
+## Expectation
+
+For each test target that includes at least one of the [visual reference words][], one of the following is true:
+
+- (**non-visual reference**) the test target is on the same [web page][] as a [textual][text] instruction that also identifies that [web content][] by a non-visual characteristic; or
+- (**visible words**) each [visual reference word][] in the test target is included in the [visible text content][] of the identified content; or
+- (**accessible words**) each [visual reference word][] in the test target is included in the [accessible name][] of the identified content; or
+- (**no indication**) the test target does not identify any [web content][] through the use of any [visual reference words][]; or
+- (**no instruction**) the test target does not give instructions about any [web content][] through the use of any of the [visual reference words][].
+
+## Assumptions
+
+- This rule assumes that [visual reference words][] are forms of information conveyed through presentation, because of this, failing this rule fails both [Success Criterion 1.3.1 Info and Relationships][sc131] and [Success Criterion 1.3.3 Sensory Characteristics][sc133]. Presentation is not limited to CSS and includes images such as the image of a circle with text.
```

Comment:
```suggestion
- This rule assumes that [visual reference words][] are forms of information conveyed through presentation. Because of this, failing this rule fails both [Success Criterion 1.3.1 Info and Relationships][sc131] and [Success Criterion 1.3.3 Sensory Characteristics][sc133]. Presentation is not limited to CSS and includes images such as the image of a circle with text.
```